resp2_replies.json

```diff
@@ -62,6 +62,45 @@
   "AUTH": [
     "[Simple string reply](../topics/protocol.md#simple-strings): `OK`, or an error if the password, or username/password pair, is invalid."
   ],
+  "BF.ADD": [
+    "One of the following:",
+    "* [Integer reply](../topics/protocol.md#integers): `1` if the item was successfully added",
+    "* [Integer reply](../topics/protocol.md#integers): `0` if the item already existed in the bloom filter",
+    "",
+    "The command may fail with an error when attempting to add to a non scaling filter that is full"
+  ],
+  "BF.CARD": [
+    "[Integer reply](../topics/protocol.md#integers): The number of items successfully added to the bloom filter, or 0 if the key does not exist"
+  ],
+  "BF.EXISTS": [
+    "One of the following:",
+    "* [Integer reply](../topics/protocol.md#integers): `1` if the item exists in the bloom filter",
+    "* [Integer reply](../topics/protocol.md#integers): `0` if the bloom filter does not exist or the item has not been added to the bloom filter"
+  ],
+  "BF.INFO": [
+    "When no optional arguments are provided:",
+    "* [Array reply](../topics/protocol.md#arrays): List of information about the bloom filter.",
+    "When an optional argument excluding ERROR is provided:",
+    "* [Integer reply](../topics/protocol.md#integers): argument value",
+    "When ERROR is provided as an optional argument:",
+    "* [String reply](../topics/protocol.md#simple-strings): argument value"
+  ],
+  "BF.INSERT": [
+    "[Array reply](../topics/protocol.md#arrays): Array of ints (1’s and 0’s) - if filter already exists or if creation was successful. An empty array if no items are provided",
+    "",
+    "The command may fail with an error for several reasons: if the provided VALIDATESCALETO is not possible, if the provided optional args are not valid"
+  ],
+  "BF.MADD": [
+    "[Array reply](../topics/protocol.md#arrays): Array of ints (1’s and 0’s)"
```

Comment:
You can add the error cases for BF.MADD: 

"The command will be rejected if input is invalid, if a non bloom filter key with the same name already exists, if the bloom filter creation / scale out exceeds limits, or if an item is being added to a full non scaling filter."